### DISCUSSION TITLE
Setting replication inside of `seed.sql` for local development

### DISCUSSION THREAD

**Discussion by ydennisy** [supabase/supabase#17585] [Questions] [answered]
Hi!

Each time we boot a fresh supabase DB, we need to access the UI and toggle replication for a few tables.

I was hoping this can be done in a startup script - ideally using `sql` in `seed.sql` :)

Here is the doc, which suggests this step is done in the UI: https://supabase.com/docs/guides/realtime/postgres-changes#quick-start

**Comment by GaryAustin1** [comment, marked as answer]
https://supabase.com/docs/guides/realtime/subscribing-to-database-changes#setup
Shows setting up replication in SQL.

**Reply by ydennisy** [reply]
Thanks very much @GaryAustin1 :)

Would it make sense to consolidate the two docs or cross link?

**Reply by GaryAustin1** [reply]
I don’t know why Supabase chose two sections.  You could suggest a change to their docs in Supabase/Supabase issues.
